source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/open-source/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+
+import { withTranslations, i18nProps } from '@lib/i18n';
+
+import { MIT } from './licenses/links';
+
+export const pathName = '/open-source';
+
+@withTranslations
+export default class OpenSourceRoute extends React.Component<i18nProps> {
+  render() {
+    const { t } = this.props;
+
+    return (
+      <div className='flex-column align-items-center w-100'>
+        <h1 className='m-t-lg'>{t('attributions.title')}</h1>
+        <hr className='w-100' />
+
+        <div className='m-t-md'>
+          <h2>{t('attributions.subtitle')}</h2>
+
+          <h3><MIT /></h3>
+          <a href='https://reactjs.org/' target='_blank'>
+            React
```

Comment:
could be in the translation file

Reply:
React is a proper noun though?

Does it have a different name in spanish?

Reply:
Codigo libre

Reply:
al though many uses open source too

Reply:
`Código libre` == `React`?